src/blog/_posts/2023-07-10-cadence-bad-practices-part-1.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Bad practices and Anti-patterns with Cadence (Part 1)
+
+date: 2023-07-10
+author: Chris Qin
+authorlink: https://www.linkedin.com/in/chrisqin0610/
+---
+
+In the upcoming blog series, we will delve into a discussion about common bad practices and anti-patterns related to Cadence. As diverse teams often encounter distinct business use cases, it becomes imperative to address the most frequently reported issues in Cadence workflows. To provide valuable insights and guidance, the Cadence team has meticulously compiled these common challenges based on customer feedback.
+
+* Reuse same workflow ID for very active/continuous running workflows
```

Comment:
This sounds a little odd. 'Reusing the same workflow ID ....'  would work better

Reply:
Done